artiq/test/lit/embedding/index_tuple.py

```diff
@@ -0,0 +1,16 @@
+# RUN: env ARTIQ_DUMP_LLVM=%t %python -m artiq.compiler.testbench.embedding +compile %s
+# RUN: OutputCheck %s --file-to-check=%t.ll
+
+from artiq.language.core import *
+from artiq.language.types import *
+
+# CHECK: void @_Z16testbench.unpackzz\(\{ i32, \{ i8\*, i32 \}, i1 \} %ARG.x\)
```

Comment:
I think it could be just ``CHECK-L`` (and with ``CHECK`` the dot after ``%ARG`` at the end is any character - not that it really matters)